base_miner/deepfake_detectors/ucf_detector.py

```diff
@@ -133,7 +130,7 @@ def infer(self, image_tensor):
         """ Perform inference using the model. """
         with torch.no_grad():
             self.model({'image': image_tensor}, inference=True)
-        return np.asarray(self.model.prob)
+        return self.model.prob[-1]
```

Comment:
model prob is a growing list of UCF's predictions, accordingly we need to return the -1th element here

Reply:
Good catch, intended to be `np.asarray(self.model.prob[-1])` to standardize numpu ndarray as the output format for UCF and NPR. Not critical

Reply:
Will look into the return types ty!